README.md

```diff
@@ -11,11 +11,8 @@ This is to ensure that the example uses both QIX engine instances and that sessi
 ## Description
 
 This example will use a [Persistent Volume](https://kubernetes.io/docs/concepts/storage/persistent-volumes/) in `Kubernetes`,
-which will be shared between the two Qlik Associative Engine instances. The synchronization between the Qlik Associative Engines will be handled by a message
-broker called [RabbitMQ](https://www.rabbitmq.com/). When a document is updated e.g. data has been reloaded or new documents were created,
-events will be placed in a queue that both instances are subscribing to.
-
-Further details on configuration of `RabbitMQ` and synchronization of documents can be found [here](https://ca.qliktive.com/docs/master/docs/services/qix-engine/doc-synchronization/).
+which will be shared between the two Qlik Associative Engine instances. The synchronization between the Qlik Associative Engines will be handled by file polling of the document on the share.
+If the data has been changed in a document existing sessions towards the document in all Qlik Associative Engine instances should be updated.
```

Comment:
What is `data`, the `data blob` (e.g. only reloads) or any data stored in the QVF? Clarify? `All existing sessions on all engine instances will be updated with new changes to the QVF file, but is not guaranteed to be instant.`?